### ISSUE TITLE
Integration of YouTube-playlists

### ISSUE THREAD

**Issue by leftshift** [gnumdk/lollypop#803]
Lollypop is about albums, designed for album playback.
Playlists are presents for later usage (phone sync, specific party, ...)
If you really need a player designed around playlists, Lollypop is a bad choice ;)

TRANSLATORS:
https://www.transifex.com/gnumdk/lollypop

WONTFIX:
- Showing bitrate

-Lollypop version:
-Gtk Version:
-Linux distribution:

-Describe your bug here:
Clearly a feature request or something I could implement, just wanted to hear your opinion on this:
I personally listen to a lot of music through youtube playlists. I use mpsyt a lot for that reason, and have just recently stumbled upon lollypop. I have to say, this looks very promising, a lot better than gnome-music.

Okay, let's get to the point: what do you think about the feature to add (dynamic) playlists from a youtube url? I'd suggest doing this in a similar fashion to "Charts"; listening to something won't add it to your library automatically but giving users that choice.

**Comment by gnumdk** [comment]
I do not understand...

**Comment by leftshift** [comment]
Okay, let me try again:
What I meant was a feature to add a youtube playlist (like https://www.youtube.com/playlist?list=UU3ifTl5zKiCAhHIBQYcaTeg) as a playlist in Lollypop

**Comment by gnumdk** [comment]
Ok, possible but not sure about syncing with youtube...

**Comment by gnumdk** [comment]
In fact, need to close this, not possible at all:
- Lollypop doesn't use youtube to get metadata because youtube API is useless for this (and youtube doesn't provide metadata, just a title)
- Lollypop get charts from Itunes/Spotify
- No way to make this smoothly integrated in UI
- Lollypop is not designed for youtube playback, it's just an extra feature and when it becomes possible (web api limitation), main backend will be Spotify

So wontfix, sorry.

**Comment by leftshift** [comment]
Okay, thanks for taking a look anyways :D
